requirements.txt

```diff
@@ -1,5 +1,5 @@
 beautifulsoup4
-Django<4
```

Comment:
Do we require anything that Django 5 introduced? If not, I would skip this change for now as it has a cascading effect on requirements in our standard WebApp configuration to Python version (easy) and PostgreSQL (not easy).

Reply:
Okay, thanks. I may check Django 4 then because it seems to still support Python 3.8

Reply:
Didn't run any tests but this branch works as-is with django 4.2.

Reply:
Confirmed. We still need to take a look at migrations and regression tests.